Content.Server/GameObjects/EntitySystems/HandsSystem.cs

```diff
@@ -48,14 +48,19 @@ public override void Initialize()
                 .Register<HandsSystem>();
         }
 
+        private void HandleExamined(EntityUid uid, HandsComponent component, ExaminedEvent args)
```

Comment:
Move this to after Shutdown so that the method order is a bit better and it should be good to merge.

Reply:
Done in: https://github.com/space-wizards/space-station-14/pull/4160/commits/0698f814fd6bc9b0906f3c0c697db1413047e0d1